zerver/lib/export.py

```diff
@@ -395,6 +405,12 @@ def write_records_json_file(output_dir: str, records: List[Dict[str, Any]]) -> N
     logging.info(" (done writing)")
 
 
+def write_message_partial(message_filename: Path, output: MessageOutput) -> None:
+    assert message_filename.endswith(".partial")
+    logging.info("%s is part of our batch processing for Message/UserMessage", message_filename)
+    write_data_to_file(output_file=message_filename, data=output)
```

Comment:
The output looks like this:

```
2021-12-08 22:40:58.402 INFO [] Exporting .partial files messages
2021-12-08 22:40:58.432 INFO [] Fetched messages for /tmp/zulip-export-cfmdwaoa/messages-000001.json.partial
2021-12-08 22:40:58.433 INFO [] /tmp/zulip-export-cfmdwaoa/messages-000001.json.partial is part of our batch processing for Message/UserMessage
2021-12-08 22:40:58.433 INFO [] Writing to /tmp/zulip-export-cfmdwaoa/messages-000001.json.partial
2021-12-08 22:40:58.433 INFO []  (done writing)
2021-12-08 22:40:58.435 INFO [] 109 messages were exported
```

I'm not sure if that's out of order (it's possible we should add flushing logic?), but I think this log line doesn't feel helpful, though we could perhaps expand `Exporting .partial files messages` to explain a bit more.

Reply:
The messages are in order of execution.  We can just drop the one message if it's confusing.

Reply:
Oh, I see, the "109 messages were exported" line confused me; I guess they were exported to partial files, but we haven't spliced in the UserMessage records.